### PR TITLE
Move Conjugations section to Control Flow page

### PR DESCRIPTION
**Pull request by gillenhaalb**
Addressing part of issue #939. Moves the Conjugations section (`apply-within` statements) from the Operations/Functions page to the Control Flow page (and adds a few lines of context on the Control Flow page)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/using-qsharp/control-flow.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7bc9c01c2b1d375722038ec5ae037112258d60ab/articles/user-guide/using-qsharp/control-flow.md) | 82 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/92200dd8-ea3d-41a8-829d-a81440748836) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/operations-functions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7bc9c01c2b1d375722038ec5ae037112258d60ab/articles/user-guide/using-qsharp/operations-functions.md) | 83 | 65 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/af67f85a-5929-4fd9-8ac9-1587a0c0e9d4) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/user-guide/using-qsharp/control-flow.md [hunk 1]
```diff
@@ -161,6 +162,47 @@ while (index < Length(arr) && item < 0) {
...
+    target : 'T) 
+: Unit {
+
+    outerOperation(target);
+    innerOperation(target);
+    Adjoint outerOperation(target);
+}
+```
+
+Starting with our 0.9 release, Q# supports a conjugation statement that implements the preceding transformation. Using that statement, the operation `ApplyWith` can be implemented in the following way:
```
Comment (geduardo) [inline articles/user-guide/using-qsharp/control-flow.md:184]:
```suggestion
Q# supports a conjugation statement that implements the preceding transformation. Using that statement, the operation `ApplyWith` can be implemented in the following way:
```

File: articles/user-guide/using-qsharp/control-flow.md [hunk 2]
```diff
@@ -161,6 +162,47 @@ while (index < Length(arr) && item < 0) {
...
+
+    within{ 
+        outerOperation(target);
+    }
+    apply {
+        innerOperation(target);
+    }
+}
+```
+Such a conjugation statement becomes far more useful if the outer and inner transformations are not readily available as operations but are instead more convenient to describe by a block consisting of several statements.
```
Comment (geduardo) [inline articles/user-guide/using-qsharp/control-flow.md:201]:
```suggestion
Such a conjugation statement becomes useful if the outer and inner transformations are not readily available as operations but are instead more convenient to describe by a block consisting of several statements. 
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/using-qsharp/control-flow.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f594ca72e1c6e73a1cf75a12f1ba1558378c94e0/articles/user-guide/using-qsharp/control-flow.md) | 82 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/410a71c5-91ac-453a-8261-953d4d7dd656) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/operations-functions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f594ca72e1c6e73a1cf75a12f1ba1558378c94e0/articles/user-guide/using-qsharp/operations-functions.md) | 83 | 65 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/47eb9768-9f56-4523-b068-46e4a4fcaa94) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/user-guide/using-qsharp/control-flow.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f594ca72e1c6e73a1cf75a12f1ba1558378c94e0/articles/user-guide/using-qsharp/control-flow.md) | 82 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/720fded0-b8ce-4d73-b1ca-6c17cd0c6b2d) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/operations-functions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f594ca72e1c6e73a1cf75a12f1ba1558378c94e0/articles/user-guide/using-qsharp/operations-functions.md) | 83 | 65 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7e06e5a1-3ddc-4641-9ac1-39a582bf578d) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by geduardo — APPROVED** [review]

**Pull request merged**